src/wagtailmedia/forms.py

```diff
@@ -29,6 +29,14 @@ def formfield_for_dbfield(db_field, **kwargs):
     return db_field.formfield(**kwargs)
 
 
+def format_extensions_for_accept_value(allowed_extensions: list[str]) -> str:
+    """Returns the specified extensions in a format usable in the `accept=""` attribute of a `FileInput` widget.
+    This assumes the list of extensions are the bare extensions e.g., `["mp4", "webm"]` and prefixes each extension with
+    a ".".
+    """
```

Comment:
non-blocking suggestion: add a link to https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/file#unique_file_type_specifiers